Sources/ISO8601DurationFormatter/DateComponents+ISO8601Duration.swift

```diff
@@ -28,15 +28,39 @@ extension DateComponents {
     - returns: A [String](https://developer.apple.com/documentation/swift/string) object [ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations) format using the current instance of [DateComponents](https://developer.apple.com/documentation/foundation/datecomponents)
      */
     public func toISO8601Duration() -> String {
```

Comment:
I think it would to introduce a parameter where the user of the library can change the way non existing values are handled. With a parameter like this. I think of users, that maybe depend on the output having 0 for non existing values.